### PR TITLE
Add text to Index page

### PR DESCRIPTION
**Pull request by ThatRendle**
Added some content to the index.md page.

### CODE REVIEW

**Comment by mairaw** [comment]
@markrendle I changed the base to master since we already merged the grpc e-book. But it seems this PR has an unwanted file?

**Comment by markrendle** [comment]
> 
> 
> @markrendle I changed the base to master since we already merged the grpc e-book. But it seems this PR has an unwanted file?

@mairaw Something really weird is happening. Trying to fix it now.

File: docs/architecture/grpc-for-wcf-developers/Protocol Buffers.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+| C# Type | Well Known Type wrapper     |
+| ------- | --------------------------- |
+| double? | google.protobuf.DoubleValue |
+| float?  | google.protobuf.FloatValue  |
+| int?    | google.protobuf.Int32Value  |
+| long?   | google.protobuf.Int64Value  |
+| uint?   | google.protobuf.UInt32Value |
+| ulong?  | google.protobuf.UInt64Value |
+
+## ??? Decimals ???
```
Comment (shirhatti) [inline docs/architecture/grpc-for-wcf-developers/Protocol Buffers.md]: Is this PR still a WIP? Or do you want to omit this for now?

**Comment by mairaw** [comment]
Do you want me to create the PR for you?

**Comment by mairaw** [comment]
Do you want to reopen this and just delete that extra file @markrendle?

File: docs/architecture/grpc-for-wcf-developers/Protocol Buffers.md (hunk 1 above)
Comment (markrendle) [reply docs/architecture/grpc-for-wcf-developers/Protocol Buffers.md]: Leave it for now. There's something up with git, I'm going to have to completely refork the repo.

**Pull request closed without merging**